### PR TITLE
chore: rename optimize 8 latest tag

### PR DESCRIPTION
**Pull request by OmranAbazid**
related to https://github.com/camunda/camunda-optimize/issues/13498

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by OmranAbazid** [comment]
There are few other `latest` docker tags that are used in the migration guide. 
I will address those in https://github.com/camunda/camunda-optimize/issues/14024

**Comment by akeller** [comment]
Is there a broader impact on the user experience with this change? Should we also include it on the [Announcements page](https://docs.camunda.io/docs/next/reference/announcements/)? I'm concerned this is a breaking change.

File: optimize/self-managed/optimize-deployment/migration-update/3.13_8.5-to-3.14_8.6.md
Comment (conceptualshark) [inline optimize/self-managed/optimize-deployment/migration-update/3.13_8.5-to-3.14_8.6.md:1]: This PR looks good to me, but is there any reason this page isn't yet added to the sidebar?

File: docs/reference/announcements.md [hunk 1]
```diff
@@ -119,6 +119,22 @@ An example of how to use the new `CorrelationResult` can be found in the [Connec
...
+
+- **Before Camunda 8.6**: Versions like `8.x` and `3.x` (used for Camunda 7) could sometimes be used interchangeably.
+- **From Camunda 8.6 onwards**: `8.6 != 3.14`. Each version corresponds strictly to its platform:
+  - **Camunda 7**: Uses the `3.x` versioning scheme and the `latest` Docker tag.
+  - **Camunda 8**: Uses the `8.x` versioning scheme and the `8-latest` Docker tag.
+
+#### Action required:
+
+- **Camunda 7 Users**: Continue using `3.x` versions and the `latest` Docker tag.
+- **Camunda 8 Users**: Update your configurations to use `8.x` versions and the `8-latest` Docker tag.
```
Comment (RomanJRW) [inline docs/reference/announcements.md:134]:
```suggestion
- **Camunda 8 Users**: If you haven't already done so, update your configurations to use `8.x` versions and the `8-latest` Docker tag.
```

@conceptualshark - do you think this makes things clearer? Note that the 8 tags have been available for some time, there has just been no guarantee that C8 customers will have been using them

File: optimize_versioned_docs/version-3.10.0/self-managed/optimize-deployment/migration-update/instructions.md [hunk 2]
```diff
@@ -107,11 +107,15 @@ However, as this may prolong the container boot time significantly which may con
...
+       # Use the appropriate image tag depending on your version
+       image: registry.camunda.cloud/optimize-ee/optimize:latest  # For Camunda 7
+       # image: registry.camunda.cloud/optimize-ee/optimize:8-latest  # For Camunda 8
        command: ['./upgrade/upgrade.sh', '--skip-warning']
    containers:
      - name: optimize
-       image: registry.camunda.cloud/optimize-ee/optimize:latest
+       # Use the appropriate image tag depending on your version
+       image: registry.camunda.cloud/optimize-ee/optimize:latest  # For Camunda 7
+       # image: registry.camunda.cloud/optimize-ee/optimize:8-latest  # For Camunda 8
```
Comment (RomanJRW) [inline optimize_versioned_docs/version-3.10.0/self-managed/optimize-deployment/migration-update/instructions.md:118]: maybe we should default to Camunda 8 here? So C7 would be the one commented out

**Review by RomanJRW — DISMISSED** [review]

File: optimize/self-managed/optimize-deployment/migration-update/3.13_8.5-to-3.14_8.6.md
Comment (OmranAbazid) [reply optimize/self-managed/optimize-deployment/migration-update/3.13_8.5-to-3.14_8.6.md:1]: I added it to the sidebar in this PR:
https://github.com/camunda/camunda-docs/pull/4241

File: docs/reference/announcements.md (hunk 1 above)
Comment (conceptualshark) [reply docs/reference/announcements.md:134]: 👍 This works for me!

**Comment by OmranAbazid** [comment]
@camunda/tech-writers Can you please approve it so that i can merge it.

**Review by conceptualshark — APPROVED** [review]

**Pull request merged**